### DISCUSSION TITLE
the car is too slippery

### DISCUSSION THREAD

**Discussion by oleale162** [igorciz777/RBC1GP-Translation-Pipeline#7] [Ideas]
thank you very much in advance, for translating it into english. this game is very hard, the car is too slippery, is it possible to put TXRD2 engine physics into this game. thank you very much

**Comment by igorciz777** [comment]
I don't think anyone's insane enough in txr community to pull that off :D It would be easier to reverse the entire game's code at that point -and we don't even have debug symbols for these games lol

Anyway, coming from a simracing mindset, I find rbc1 to be way more controllable than d2.
Play around with the setups more, maybe wait for the full BBS translation update to drop, since that has a lot of hints for car setup?

**Reply by oleale162** [reply]
i think you are right, usually i play racing video games just by choosing a car, then start racing, without knowing about car setup, and it works in richard burns rally game. maybe genki wants to include the fun element of car tuning in the game, i will try to find out on the internet how to tune a car in this game. thank you very much for translating it into english
